### PR TITLE
[IBM] Mainly style related corrections to two how-to guides

### PR DESCRIPTION
**Pull request by k-dimple**

### CODE REVIEW

File: ibm/ibm-how-to/launch-ubuntu-instances.rst [hunk 1]
```diff
@@ -1,125 +1,107 @@
...
 
-See https://cloud.ibm.com/docs/vpc?topic=vpc-profiles&interface=ui for information on selecting a profile.
+Replace ``<vpc-id>``, ``<zone>``, ``<image-id>`` and ``<ssh-key-id>`` with the information gathered above. If you need help deciding on the instance-type refer to `IBM's documentation on instance profiles`_.
 
-A filled out example command would look like this:
+A complete example command would look like this:
 
 .. code-block:: bash
 
    ibmcloud is instance-create TODO: fill in the rest of the command
```
Comment (k-dimple) [inline ibm/ibm-how-to/launch-ubuntu-instances.rst:102]: @a-dubs could you fill this in? We missed it in our last round of reviews.

Comment (a-dubs) [reply ibm/ibm-how-to/launch-ubuntu-instances.rst:102]: AH RIGHT OOPS

the image id used for this command (r006-3a44e4ee-9c9f-4693-98ae-fced7a46ffce) is for Ubuntu 24.04 LTS.
```
ibmcloud is instance-create MyUbuntuInstance \ 
    r006-613dd231-0123-4567-8901-1de1ce8116fa us-south-1 bx2-2x8 r006-3a44e4ee-9c9f-4693-98ae-fced7a46ffce \
    --keys r006-b1fd6d2c-abcd-ef12-3456-4f44d2e16aac 
```

**Review by a-dubs — CHANGES_REQUESTED** [review]
Looks great @k-dimple! Thank you for going through and cleaning this up. Much appreciated. 

Gonna leave as "requested changes" just so we don't forget to fill in that TODO again haha

**Comment by k-dimple** [comment]
Thanks @a-dubs! I have included the command (but on second thoughts I decided to hide the actual VPC ID and SSH key).

**Comment by a-dubs** [comment]
@k-dimple hahaha alright sounds good. I did scramble the IDs though so they were bogus values, but in the right format

**Comment by k-dimple** [comment]
> @k-dimple hahaha alright sounds good. I did scramble the IDs though so they were bogus values, but in the right format

ah!

**Pull request merged**